.github/workflows/android.yml

```diff
@@ -35,10 +35,13 @@ jobs:
       endsWith(github.ref, '/next') &&
       !contains(github.event.head_commit.message, 'release prod')
     steps:
+      - name: Pull dev vercel secrets
+        if: ${{ !endsWith(github.ref, '/next') }}
+        run: vercel pull --yes --environment=preview --git-branch=${{ github.ref_name }} --token=${{ secrets.VERCEL_TOKEN }}
```

Comment:
next - environment=production
staging - environment=preview git-branch=staging
master - environment=preview

Reply:
i'm not sure what will happen if you do envrionment=preview and git-branch=master
have you test it locally?

Reply:
yes I ran the cli commands, so --environment=preview --git-branch=master equals to pulling in dev-env vars